build.gradle

```diff
@@ -26,12 +38,29 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.5.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.5.2'
     testImplementation 'org.junit.jupiter:junit-jupiter-params:5.5.2'
-    testImplementation 'com.opentable.components:otj-pg-embedded:0.13.3'
     testImplementation 'org.hamcrest:hamcrest:2.2'
     testImplementation 'ch.qos.logback:logback-classic:1.2.3'
     testImplementation 'org.mockito:mockito-core:3.2.0'
-    testImplementation 'io.zonky.test:embedded-postgres:1.2.6'
-    testImplementation enforcedPlatform('io.zonky.test.postgres:embedded-postgres-binaries-bom:11.5.0')
+
+    pickPostgres(
+            [
+                    'com.opentable.components:otj-pg-embedded:0.13.3'
```

Comment:
I'd prefer to keep 'io.zonky.test' dependency because it allows to change PG version easily.